### ISSUE TITLE
Feature: Ability to undeploy an SDF project

### ISSUE THREAD

**Issue by mayerlench** [oracle/netsuite-suitecloud-sdk#283]

**Comment by alii-gator** [comment]
@mayerlench Can you please give any use cases for this?
Are you looking for the ability for ACP projects or SuiteApp projects or both?

**Comment by mayerlench** [comment]
> @mayerlench Can you please give any use cases for this?
> Are you looking for the ability for ACP projects or SuiteApp projects or both?
> 

Yes, if I deploy an SDF account customization that has say 10 objects. 

There's no way currently to undeploy, I would have to log in to netsuite and delete each object

**Comment by alii-gator** [comment]
@mayerlench Currently NetSuite only supports uninstalling of a SuiteApp from Account. It's not yet supported with the tool.
I will discuss this with the team and probably you will have to request this as a Feature Enhancement  by contacting NetSuite Support.

**Comment by Montsant19** [comment]
Internally tracked but not completely depending on our team.
